userprofile/static/userprofile/copy_token.js

```diff
@@ -0,0 +1,21 @@
+function copyToken(text, el) {
```

Comment:
There is at least a second similar script in A+. Could we share the implementation to multiple locations? I.e. could there be a "copy-to-clipboard" component?

Reply:
https://github.com/apluslms/a-plus/blob/32b86bf17a33a5c001cc4a3ac87c41b50d8e4100/assets/js/aplus.js#L179
This JS has been used in the submitted file modal dialog in order to copy the file contents to the clipboard.

Reply:
Actually, A+ uses https://clipboardjs.com/. Therefore, we can make use of this library in different locations.

Reply:
Anycase, I will create a **copy-to-clipboard.js** file with a more generic implementation.